benchmark/streams/pipe-unpipe.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const common = require('../common');
+const { Readable, Writable } = require('stream');
+
+const bench = common.createBenchmark(main, {
+  n: [5e6]
+});
+
+function main({ n }) {
+  const b = Buffer.alloc(1024);
+  const r = new Readable();
+  const w = new Writable();
+
+  var i = 0;
+
+  r._read = () => r.push(i++ === n ? null : b);
+  w._write = (data, enc, cb) => cb();
+
+  bench.start();
+
+  r.pipe(w);
+  r.on('data', () => {
+    r.unpipe();
+  });
+  w.on('unpipe', () => {
+    bench.end(n);
+  });
+}
```

Comment:
Is this benchmark useful in general or is it very specific to this change? If it's generic, I'd keep it and otherwise remove it again.

Reply:
I'd like to remove it